Contribute/powershell-contribute.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Contributing to PowerShell documentation
+description: This article is an overview of how to get started as a contributor to the PowerShell documentation.
+ms.date: 03/05/2020
+ms.topic: contributor-guide
+ms.prod: non-product-specific
+ms.custom: external-contributor-guide
+author: sdwheeler
+ms.author: sewhee
+---
+# Contributing to PowerShell documentation
+
+Thank you for your support of PowerShell!
+
+The Contributor's Guide is a collection of articles that explain the tools and processes we use to
+create documentation at Microsoft. Some of these guides cover information that is common to any
+documentation set published to [docs.microsoft.com][docs]. The PowerShell documentation contains
+[guides][psdocs] that are specific to how we write documentation for PowerShell.
+
+The common articles are available in this centralized Contributor's Guide. The PowerShell-specific
```

Comment:
One suggestion for this PR. What do you think about this sentence "The common articles..." being split into the two lines that incorporate the links to the guides (versus the sentences followed by the links after)?

Reply:
I split this up.